CHANGELOG.md

```diff
@@ -16,13 +16,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [2.1.0] 2022-06-19
```

Comment:
We prob don't want to bump major v, but I highly recommend you add a setting saying "USE_HASHED_VALUES=True" or something like that and set that to false by default. Users cannot migrate unless that setting value is true. If not, raise an error during migration.

I've had poor experiences with breaking changes being hidden in minor versions, and many people don't unit test this pkg , so highly recommend

Reply:
The breaking change happened with 2.0.0 with hashing and PKCE required. There's no breaking change w.r.t hashing here -- a bugfix for migrations which would have been a patch release (2.0.1) but then I included a new backwards-compatible feature (prompting) which resulted in 2.1.0. 

Given the repeated issues raised with `{"error": "invalid_client"}` caused generally by not using the cleartext client secret or not using PKCE I decided to update the README in the hopes that people would see that before raising the issue.